src/main/java/com/ibm/etcd/client/lease/EtcdLeaseClient.java

```diff
@@ -538,7 +541,7 @@ private void revoke() {
                     //TODO convert to use common GrpcClient retry logic probably
                     ses.schedule(LeaseRecord.this::revoke, 2, SECONDS);
                 }
-            });
+            }, executor);
```

Comment:
maybe just `MoreExecutors.directExecutor()` directly here rather than additional var?

Reply:
Agree. No need to not be consistent :+1: